src/main/java/mat/client/measure/ComponentMeasureSearch.java

```diff
@@ -303,7 +303,8 @@ private void buildAppliedComponentMeasuresTableColumns() {
 			public SafeHtml getValue(ManageMeasureSearchModel.Result object) {
 				SafeHtmlBuilder sb = new SafeHtmlBuilder();
 				sb.appendHtmlConstant("<div id='appliedComponentContainer' tabindex=\"-1\">");
-				sb.appendHtmlConstant("<span id='appliedComponentDiv' title=\" " + object.getName() + "\" tabindex=\"0\">" + object.getName() + "</span>");
+				sb.appendHtmlConstant("<span id='appliedComponentDiv' title=\" " + SafeHtmlUtils.htmlEscape(object.getName()) +
```

Comment:
Can you please check if the Ids `appliedComponentDiv`, `div2` etc are really needed.  If not you can remove them or use class.

Reply:
We have CSS properties associated with these ids. Should I change these styling using 'class'?

Reply:
Done !